package.json

```diff
@@ -116,7 +116,7 @@
     "resize-observer-polyfill": "^1.5.1",
     "tailwindcss": "^3.1.8",
     "ts-standard": "11.0.0",
```

Comment:
We want to upgrade ts-standard, not Typescript (unless it's required by ts-standard).

```
yarn upgrade ts-standard@^12.0.2
```

Reply:
okay

Reply:
![image](https://github.com/OpenBeta/open-tacos/assets/81979505/fdbf3f67-23df-447c-a65e-d2c1e738f1c6)
@vnugent, if I upgrade to `ts-standard@^12.0.2`, we will need to make changes in a lot of files, and even after trying, I still get errors. What should I do?

Reply:
Some of them area valid warnings and we'll have to fix them one by one.  If you want  I can take over this PR and fix them.

Reply:
I will try. Can you please tell me which area has valid warnings?

Reply:
There's a problem:` <form onSubmit={handleSubmit(onSubmit)}>`. I am getting the error message Promise-returning function provided to the property where a void return was expected. in most of the files. However, I have solved other problems. Can you please guide me?

Reply:
@vnugent we don't have eslint rules file at the root of the project.

Reply:
![image](https://github.com/OpenBeta/open-tacos/assets/81979505/4573f774-1e64-4703-9ff9-c8f647ae0914)
Now only 8 warnings remain, and I have been trying to solve them, but I haven't found any solution yet.